webapp-django/pytest.ini

```diff
@@ -5,4 +5,12 @@
 addopts = -rsxX --tb=native
 DJANGO_SETTINGS_MODULE = crashstats.settings.test
 # Transform all warnings into errors
-filterwarnings = error
+filterwarnings =
+    error
+    # grpcio kicks this up for some reason in pytest
+    ignore::ImportWarning
+    ignore:::grpcio[.*]
+    ignore:::google[.*]
+    # configman kicks up errors if there's configuration stuff that isn't used
+    # and that breaks the webapp's configman glue
+    ignore:::configman[.*]
```

Comment:
We have pytest set up to treat warnings as errors. However, some of the libraries we use kick up warnings and we don't want to treat *those* as errors. Until they get fixed, we'll configure pytest to ignore them.